scripts/kickstart_generator.rb

```diff
@@ -43,23 +43,6 @@ def gen_file_path(file)
 
     private
 
-    def write_config(result)
-      file = @ks_gen_base.join("base-#{@target}.cfg")
-      $log.info("Writing kickstart in config format: #{file}") if $log
-      File.write(file, result)
-    end
-
-    def write_json(result)
```

Comment:
See my comments above. I've restructured `vmbuild.rb` in a separate commit in this PR. I could leave this code it but it would be unused code.